rectools/visuals/metrics_app.py

```diff
@@ -25,7 +26,10 @@
 WIDGET_WIDTH = 800
 WIDGET_HEIGHT = 500
 TOP_CHART_MARGIN = 20
-LEGEND_TITLE = "Models"
+DEFAULT_LEGEND_TITLE = "model name"
+NAN_COLOR = "grey"
+META_MODEL_SEP = "❘"
```

Comment:
Do you think it makes sense to add spaces around for readability? (If yes, then also for the next one)

Reply:
I checked both options, and personally prefer version without additional spaces. Let me show you:
Without additional spaces:
![Screenshot from 2024-07-26 10-53-05](https://github.com/user-attachments/assets/22eec3c7-4ab8-461e-9705-9b8644f264f1)
With them:
![Screenshot from 2024-07-26 10-52-30](https://github.com/user-attachments/assets/cdabc6fe-9988-4685-aeed-5330512b0f71)

Reply:
Ok, doesn't  matter choose whatever you prefer, both looks fine for me